### ISSUE TITLE
Can't get the plugin to work

### ISSUE THREAD

**Issue by mundschenk-at** [rhubarbgroup/email-address-encoder#2]
The plugin does not seem to do anything on my websites (e.g. https://mundschenk.at/). The filters are added and get called normally, but the addresses are not replaced (debug output enabled for `preg_replace_callback`). I haven't hat the time yet to debug the actual regex (although on http://www.phpliveregex.com/ it seems to work correctly). Any idea what could be going wrong?

**Comment by tillkruss** [comment]
Either the filters aren't applied to your text, or you're not checking the results the right way. Check the FAQ on that.

**Comment by mundschenk-at** [comment]
As I said, the filters were called, but the results returned by `preg_replace_callback` were unchanged. We are talking about `error_log()` output here, so telling me to read the FAQ is somewhat disingenious.

**Comment by tillkruss** [comment]
Are you seeing any errors in your logs? How are you verifying whether or not the reflex works?

**Comment by mundschenk-at** [comment]
@tillkruss Thanks for re-opening the issue. 

It appears that during the testing phase earlier today (I had to adjust filter priorities to make it work with my wp-Typography plugin), I made some mistake when I replaced the `create_function` call with a closure to prevent PHP 7.2 deprecation warnings. The perils of quickly coding something with `vi`. 

I've not tried again with a vanilla version and the plugin works intended now. Are you willing to entertain a PR regarding the `create_function` deprecation? It would necessitate requiring at least PHP 5.3, but since so few people still use PHP 5.2, I think that can easily be justified.
